dephell/imports.py

```diff
@@ -0,0 +1,50 @@
+# built-in
```

Comment:
Thanks.

BTW, I had issues with this style of lazy import once (related to access to attributes). I have solved them with a `lazy_object_proxy`, but the solution was a bit ugly, so I had to write a lib https://gitlab.com/KOLANICH/lazily.py doing it a bit nicer. But it works not very fine, causes some glitches (when importing from already imported modules), so I had to stop using it. I have already spend a lot of time on debugging it and even written an absolutely necessary tool https://gitlab.com/KOLANICH/FrozenTable.py (one cannot just patch importlib to add some printing without it), but I don't have time for it now. I guess you may be interested.